### ISSUE TITLE
Chunks don't load around the camera in 1.18 worlds.

### ISSUE THREAD

**Issue by roboticvenusian** [Amulet-Team/Amulet-Map-Editor#485]
**Has this bug already been reported? Please check if there is a bug report already open that matches this.**
Nothing specifically for 1.18 worlds.

**Describe the bug**
When loading 1.18 worlds, only the chunks around 0,0 are loaded, and once the max number in render distance is met, it stops loading any more. 

**To Reproduce**
Load a pre-populated world, mine is specifically a world used in a paper server I run, and go to anywhere past the render distance, and nothing else loads.

**Expected behavior**
When other non-1.18 worlds are loaded, the chunks load around the camera.

**Screenshots**
Below is my camera at -1318, 70, -923

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 0.8.17a0-Windows-x86

![bug capture](https://user-images.githubusercontent.com/95646422/144896340-bcc9976f-a9d9-4c64-a9d4-6bd430b2c604.PNG)

**Comment by roboticvenusian** [comment]
I just realized that it may be because the camera is still being considered at 0,0, based on the coordinates next to the 3D option.

**Comment by gentlegiantJGC** [comment]
Yeh I am not sure what happened there. Does it not update if you move around?

**Comment by roboticvenusian** [comment]
No, it just stays empty. The screenshot is me at -1318, -923 after moving there manually, and nothing updated past the chunks nearest to 0,0. The only thing that changed anything was the render distance, but even then it only increased/decreased the chunks around 0,0.

**Comment by gentlegiantJGC** [comment]
I have just realised that someone else has had this issue in the past. See #322 
Still not sure what would cause this.
